meshroom/core/graph.py

```diff
@@ -234,7 +234,15 @@ def fileFeatures(self):
         return Graph.IO.getFeaturesForVersion(self.header.get(Graph.IO.Keys.FileVersion, "0.0"))
 
     @Slot(str)
-    def load(self, filepath):
+    def load(self, filepath, fileLink=True):
+        """
+        Load a meshroom graph ".mg" file.
+
+        Args:
+            filepath: project filepath to load
+            fileLink: Setup link to the project file, like setup cacheDir, keep filepath for save, etc.
```

Comment:
```suggestion
            fileLink: if false, it only loads the project file as a template (all the data is discarded) and only the bare structure of the graph is preserved.
```
I found `Setup link to the project file, like setup cacheDir, keep filepath for save, etc.` a bit obscure/confusing